src/controller.js

```diff
@@ -121,13 +120,14 @@ class Controller {
   }
 
   async typeHackOn() {
-    console.log(this.hackOn);
     for (var i = 0; i < this.hackOn.length; i++) {
-      await this.typeCharacter(this.hackOn[i], null, 0);
-      if(i === 74) {
-        this.term.state.setCursor(0, this.term.state.cursor.y+1);
+      if(this.hackOn[i] === '\n') {
+        this.addNewLine()
+      } else {
+        await this.typeCharacter(this.hackOn[i], null, 0);
       }
     }
+    this.addNewLine()
```

Comment:
This could be  achieved by adding a new line in the string literal or could go here. Doesn't really matter but thought I would explain it.